### PR TITLE
allows http:// and other protocol identifiers to be used in namespaces

### PR DESCRIPTION
**Pull request by tomeara**
The replace that was used on the `namespacedURL` did not allow parts of the namespace string to have repeated slashes. This now just strips off any additional slashes from either the end of the namespace string, or the beginning of the handler's url string, and then joining them together. 
Please check issue #306 for more of the conversation.

_Secondary changes: the build steps updated the copyright_

### CODE REVIEW

File: dist/jquery.mockjax.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
  * 
  * Version: 2.2.1
  * Home: https://github.com/jakerella/jquery-mockjax
- * Copyright (c) 2016 Jordan Kasper, formerly appendTo;
+ * Copyright (c) 2017 Jordan Kasper, formerly appendTo;
```
Comment (jakerella) [inline dist/jquery.mockjax.js]: :) Thanks

**Comment by tomeara** [comment]
I have found an issue with this solution if the namespace is just a slash `namespace = '/'`. I have a good failing spec and I'll work to fix it.

**Comment by tomeara** [comment]
Nevermind, looks like it was something wrong with my implementation. Sorry about that.

**Comment by jakerella** [comment]
Can we add one more test with a double slash? Something like:

```
$.mockjaxSettings.namespace = '/api/';

$.mockjax({  url: '/foo', ...  });
```

**Comment by tomeara** [comment]
Just added another test using `http://localhost/` as the namespace and `/myservice` as the url.

**Comment by jakerella** [comment]
Awesome, thanks!

**Pull request merged**